src/VueSlickCarousel.vue

```diff
@@ -108,6 +108,9 @@ export default {
     this.clearBreakpoints()
   },
   methods: {
+    getCurrentSlide() {
+      return this.$refs.innerSlider.getCurrentSlide()
```

Comment:
You can just use `this.$refs.innerSlider.currentSlide` instead of adding a method inside `InnerSlider.vue`

Reply:
Also, inside the component which uses `VueSlickCarousel`, you can hit it like `this.$refs.myVueCarousel.$refs.innerSlider.currentSlide`, without having your particular fix. It's less clean, but it would work on the lastest official version I think